src/js/components/molecules/footer-navigation-category/index.ts

```diff
@@ -0,0 +1 @@
+export { FooterNavigationCategory as default } from './footer-navigation-category';
```

Comment:
How do you feel about making `FooterNavigationCategory` a default export so you can `export { default } from './footer-navigation-category';`? here?

Reply:
No changing, this is throughout the whole design system